projectGuide/easyshop-kind.md

```
@@ -110,107 +110,93 @@ This is just a preview want more details, then use `tree` inside the repo.
-```zsh
-kubectl apply -f kubernetes/04-configmap.yaml
+> ##### 5.  Setup storage
+>   ```zsh
+>   kubectl apply -f kubernetes/02-mongodb-pv.yaml
+>   kubectl apply -f kubernetes/03-mongodb-pvc.yaml
+>   ```
 
-## This is the configmap manifest.
+> ### 5. Create ConfigMap
+> Create `kubernetes/04-configmap.yaml` with the following content:
+>   ```yaml
```

markdownlint-cli2 (0.17.2)

Multiple spaces after blockquote symbol
null

(MD027, no-multiple-space-blockquote)